### PR TITLE
create_dmd_release: Copy Win32 import libraries from Druntime

### PR DESCRIPTION
**Pull request by CyberShadow**
For https://github.com/D-Programming-Language/druntime/pull/1402

Untested.

### CODE REVIEW

**Comment by MartinNowak** [comment]
Could implibs simply become part of the default target?

File: create_dmd_release/create_dmd_release.d [hunk 1]
```diff
@@ -602,6 +607,8 @@ void createRelease(string branch)
         {
             copyFile(cloneDir~"/druntime/lib/gcstub.obj", osDir~"/lib/gcstub.obj");
             copyFile(cloneDir~"/phobos/phobos.lib", osDir~"/lib/phobos.lib");
+            foreach (de; dirEntries(cloneDir~"/druntime/lib/win32/", "*.lib", SpanMode.shallow))
```
Comment (MartinNowak) [inline create_dmd_release/create_dmd_release.d]: You can use `copyDir(cloneDir~"/druntime/lib/win32/", osDir~"/lib/", file => file.endsWith(".lib"))` instead.

**Comment by CyberShadow** [comment]
> Could implibs simply become part of the default target?

Unfortunately the `implib` utility is not in the autotester's PATH.

**Comment by MartinNowak** [comment]
What's the implib tool? Do I need to install it? That might require to adjust the windows image used to build the release.

**Comment by CyberShadow** [comment]
> What's the implib tool?

Uhh, it creates Windows OMF import libraries from .def files, among other things.

> Do I need to install it?

It's distributed with DMC.

> That might require to adjust the windows image used to build the release.

Does it not have the DMC tools? How does it even build DMD? This doesn't work on the autotester because it uses absolute paths.

**Comment by MartinNowak** [comment]
Let's . How does that work with coff btw, are the files part of VC?

**Comment by CyberShadow** [comment]
Yep. Nothing to do with COFF because unsurprisingly, Microsoft's compilers provide complete import library coverage for Microsoft's OS :)

**Comment by MartinNowak** [comment]
I had to install implib, update the Windows image, and fix win32.mak (https://github.com/D-Programming-Language/druntime/pull/1438) :(.

**Comment by MartinNowak** [comment]
Should be available tomorrow morning with the new nightly.
https://builds.dawg.eu/

**Pull request merged**